src/stactools/esa_worldcover/constants.py

```diff
@@ -150,18 +151,26 @@
             "University"
         ),
         roles=[ProviderRole.PROCESSOR],
-        url="https://worldcover2020.esa.int/",
+        url="https://esa-worldcover.org/en",
```

Comment:
```suggestion
        url="https://esa-worldcover.org",
```

No reason to pick a language for folks, right?